x-pack/examples/third_party_vis_lens_example/public/visualization.tsx

```diff
@@ -75,7 +75,7 @@ export const getRotatingNumberVisualization = ({
   },
 
   getSuggestions: ({ state, table }) => {
-    if (table.columns.length > 1) {
+    if (!table.columns.length || table.columns.length > 1) {
```

Comment:
Is this equivalent to `table.columns.length > 0`?

Reply:
no, 2 cases where it's not equivalent:` an empty array` and `an array with 1 element`. 
Looks like here we tried to check for `table.columns.length !==  1`